docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
+What will you get?
+------------------
+[video-placeholder]
 
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :alt: PyTorchLightning neptune.ai integration
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+* preview running experiment
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
 
-.. code-block:: bash
+.. tip::
```

Comment:
I think we should have 3 things close to the top (perhpas the tip or note are great options).

- link to project in Neptune
- link to full code that runs it (so that you know there is some code that is tested)
- link to colab if applicable (just so you know you can) same as full code

Reply:
corrected.